apps/daimo-mobile/src/logic/log.ts

```diff
@@ -5,6 +5,16 @@ interface LogAction {
   error?: string;
 }
 
+export class NamedError extends Error {
+  constructor(message: string, public name: string) {
+    super(message);
+
+    // Set the prototype explicitly.
+    // https://github.com/microsoft/TypeScript-wiki/blob/81fe7b91664de43c02ea209492ec1cea7f3661d0/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work
+    Object.setPrototypeOf(this, NamedError.prototype);
```

Comment:
just curious, what was the issue that this fixed?

Reply:
no observed issue but other code online recommended adding it (and the typescript github link explains why)